1.Exploratory_Data_Analysis/notebooks/README.md

```diff
@@ -0,0 +1,23 @@
+# Exploratory data analysis
+
+This folder contains notebooks that demonstrate how to perform exploratory data analysis (EDA) on a dataset.
+EDA is an essential step in the data analysis process.
+It allows you to summarize the main characteristics of the data, often with visual methods. This helps you understand the data and the relationships between the variables.
+
+Due to the exploratory nature of the analysis, some analysis are secondary and were not included in the manuscript that this repository is for.
+However, they are still useful for understanding the data and the relationships between the variables.
+
+The main analysis that are needed for the manuscript are:
+
+1. [UMAP](1.umap_analysis_plate2.ipynb)
```

Comment:
consider a brief blurb describing what "plate 2" data are. Alternatively, you can avoid saying "plate 2" here and just state directly what plate 2 is.

Reply:
Adding link to the repo!